Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -218,8 +218,8 @@ extension SwiftLanguageService {
     }
 
     if case .dictionary(let experimentalCapabilities) = self.capabilityRegistry.clientCapabilities.experimental,
-      case .bool(true) = experimentalCapabilities["workspace/peekDocuments"],
-      case .bool(true) = experimentalCapabilities["workspace/getReferenceDocument"]
+      case .bool(true) = experimentalCapabilities["workspace/peekDocuments"]
+      // TODO: Check if client supports LSP 3.18's workspace/textDocumentContent
```

Comment:
I guess this check should be in place before we merge the PR

Reply:
Could you address this TODO? I don’t like having TODOs in the codebase without an associated issue because in practice they never get fixed.

Reply:
I've re-added this and added a short remark to the PR description for how this check has to be updated once we wish to support the upstream client capability.